config/configuration.yaml

```diff
@@ -2,7 +2,8 @@
 mqtt_settings:
   host: mosquitto
   port: 1883
-  command_topic: commands/#  # The mqtt topic that contains the relevant commands.
+  command_topic: commands/#  # We subscribe to this MQTT topic to receive commands
+  error_topic: error  # If this has a value, we publish error messages under this MQTT topic
```

Comment:
we will probably need to update the README as well. Feel free to that as a separate task though